packages/shared/src/components/RecommendedMention.tsx

```diff
@@ -0,0 +1,48 @@
+import classNames from 'classnames';
+import React, { ReactElement } from 'react';
+import { UserShortInfo } from './profile/UserShortInfo';
+
+interface RecommendedUser {
+  image: string;
+  name: string;
+  username: string;
+}
+
+interface RecommendedMentionProps {
+  users: RecommendedUser[];
+  selected: number;
+  onClick?: (username: string) => unknown;
+}
+
+export function RecommendedMention({
+  users,
+  selected,
+  onClick,
+}: RecommendedMentionProps): ReactElement {
+  if (!users?.length) {
+    return null;
+  }
+
+  return (
+    <ul
```

Comment:
I'm not sure what should be the accessibility aspects of this component but you can have a look at Twitter and see if they added any attributes or keyboard behavior, etc. Please let me know your findings before you implement

Reply:
The noticeable thing on their structure looks to be utilizing a role attribute as `listbox` (select element) for the container and `option` role for each item. Then an aria attribute `aria-selected` for which one is currently selected (which now I think a must, to implement).

Reply:
For now, I will just apply the `aria-selected` attribute as it seems to be a must for this scenario. What do you think?

Reply:
I think the roles are also important so apply them as well, please.
They turn this component into a sort of input component, dropdown to be more accurate. So it's important to set these.